pages/dao/[symbol]/proposal/components/instructions/CreateAssociatedTokenAccount.tsx

```diff
@@ -0,0 +1,174 @@
+    const isValid = await validateInstruction()
+
+    if (
+      !connection ||
+      !isValid ||
+      !programId ||
+      !form.governedAccount?.governance?.account ||
+      !form.splTokenMintUIName ||
+      !wallet?.publicKey
+    ) {
+      return {
+        serializedInstruction: '',
+        isValid: false,
+        governance: form.governedAccount?.governance,
+      }
+    }
+
+    const [tx] = await createAssociatedTokenAccount(
+      // fundingAddress
+      wallet.publicKey,
+
+      // walletAddress
+      form.governedAccount.governance.pubkey,
+
+      // splTokenMintAddress
+      getSplTokenMintAddressByUIName(form.splTokenMintUIName)
+    )
+
+    console.log('infos', {
+      governanceAccount: form.governedAccount.governance.account,
+      governanceOwner: form.governedAccount.governance.owner.toString(),
+      fundingAddress: wallet.publicKey.toString(),
+      walletAddress: form.governedAccount.governance.pubkey.toString(),
+      splTokenMintName: form.splTokenMintUIName,
+      splTokenMintAddress: getSplTokenMintAddressByUIName(
+        form.splTokenMintUIName
+      ).toString(),
+    })
+
+    console.log('tx', tx)
+
+    return {
+      serializedInstruction: serializeInstructionToBase64(tx),
+      isValid: true,
+      governance: form.governedAccount.governance,
+    }
+  }
+
+  useEffect(() => {
+    handleSetForm({
+      propertyName: 'programId',
+      value: programId?.toString(),
+    })
+  }, [realmInfo?.programId])
+
+  useEffect(() => {
+    handleSetInstructions(
+      {
+        governedAccount: form.governedAccount?.governance,
+        getInstruction,
+      },
+      index
+    )
+  }, [form])
+
+  const schema = yup.object().shape({
+    governedAccount: yup
+      .object()
+      .nullable()
+      .required('Governed account is required'),
+    splTokenMintUIName: yup.string().required('SPL Token Mint is required'),
+  })
+
+  return (
+    <>
+      <GovernedAccountSelect
+        label="Governance"
+        governedAccounts={governedAccounts as GovernedMultiTypeAccount[]}
+        onChange={(value) => {
+          handleSetForm({ value, propertyName: 'governedAccount' })
+        }}
+        value={form.governedAccount}
+        error={formErrors['governedAccount']}
+        shouldBeGoverned={shouldBeGoverned}
+        governance={governance}
+      ></GovernedAccountSelect>
+
+      <Select
+        label="SPL Token Mint"
+        value={form.splTokenMintUIName}
+        placeholder="Please select..."
+        onChange={(value) =>
+          handleSetForm({ value, propertyName: 'splTokenMintUIName' })
+        }
+        error={formErrors['baseTokenName']}
+      >
+        {Object.entries(SPL_TOKENS).map(([key, { name, mint }]) => (
+          <Select.Option key={key} value={name}>
+            <div style={{ display: 'flex', flexDirection: 'column' }}>
+              <span>{name}</span>
+
+              <span style={{ color: 'grey', fontSize: '0.8em' }}>
```

Comment:
project uses tailwind if I'm not mistaken, we should follow the existing convention